airbyte_cdk/sources/declarative/models/declarative_component_schema.py

```diff
@@ -1774,6 +1800,17 @@ class HttpRequester(BaseModel):
     parameters: Optional[Dict[str, Any]] = Field(None, alias="$parameters")
 
 
+class DynamicSchemaLoader(BaseModel):
+    type: Literal["DynamicSchemaLoader"]
+    retriever: Union[AsyncRetriever, CustomRetriever, SimpleRetriever] = Field(
+        ...,
+        description="Component used to coordinate how records are extracted across stream slices and request pages.",
+        title="Retriever",
+    )
+    schema_type_identifier: SchemaTypeIdentifier
+    parameters: Optional[Dict[str, Any]] = Field(None, alias="$parameters")
+
+
```

Comment:
I think you need to add DynamicSchemaLoader to schema_loader options [here](https://github.com/airbytehq/airbyte-python-cdk/blob/ed9a5e7aafb1ba4b9be04db70003e29d4c765de9/airbyte_cdk/sources/declarative/models/declarative_component_schema.py#L1612).

Reply:
This is auto-generated file, so we don't need change manually anything here.